src/stores/WalletStore.ts

```diff
@@ -215,6 +215,10 @@ export default class WalletStore {
           this.testnetAccounts = toJS(testnetAccounts);
         }
 
+        if (networkIndex === 0 || networkIndex === 1) {
```

Comment:
can shorten this with `if (networkIndex) {`. this check should be just to see if its defined or not. the actual validation should be done before storing the value in storage.

Reply:
I considered doing that too,  but the problem with that is if networkIndex == 0, it will return false.
The other option I considered was not storing networkIndex, but instead storing a string  for which network it is on, but I think since I'm actually switching the network on networkIndex, it is better to store networkIndex directly.